src/views/DATA/DATA.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect } from 'react'
+import useDataTokenMarketData from 'hooks/useDataTokenMarketData'
+import useDataComponents from 'hooks/useDataComponents'
+import useBalances from 'hooks/useBalances'
+import { DataIndex } from 'constants/productTokens'
+import ProductDataUI, {
+  TokenDataProps,
+} from 'components/ProductPage/ProductDataUI'
+
+const DataProductPage = (props: { title: string }) => {
+  useEffect(() => {
+    document.title = props.title
+  }, [props.title])
+
+  const { latestPrice, prices, hourlyPrices, latestMarketCap, latestVolume } =
+    useDataTokenMarketData()
+  const { components } = useDataComponents()
+  const { dataBalance, dataTotalSupply } = useBalances()
+  const tokenDataProps: TokenDataProps = {
+    prices: prices,
+    hourlyPrices: hourlyPrices,
+    latestPrice: latestPrice,
+    latestMarketCap: latestMarketCap,
+    latestVolume: latestVolume,
+    token: DataIndex,
+    components: components,
+    balance: dataBalance,
+    currentSupply: dataTotalSupply,
+  }
+
+  return (
+    <ProductDataUI tokenDataProps={tokenDataProps}>
+      <p>
+        <strong>The Data Economy Index (DATA)</strong> is designed to capture
+        the trend of entertainment, sports and business shifting to take place
+        in virtual environments.
+        <h2>Constituent Weighting</h2>
+        Selection of the $DATA tokens is based on the following basic criteria:
+        <ul>
+          <li>The token must be available on the Ethereum blockchain.</li>
+          <li>
+            Protocol must be in one of the following token categories on
+            Coingecko: Non Fungible Tokens, Entertainment, Virtual Reality,
+            Augmented Reality and Music. More categories will be added in the
+            future as the market matures.
+          </li>
+          <li>Total market cap must be over $30m.</li>
+          <li>
+            Protocol must have at least 3 months history of operation and its
+            token must have at least 3 months of price and liquidity history.
+          </li>
+          <li>
+            Token must have reasonable and consistent DEX liquidity on Ethereum.
+          </li>
+          <li>
+            An independent security audit should have been performed on the
+            protocol and results reviewed by the product methodologist. In the
+            case that no audit has been performed, the methodologist applies
+            subjective judgement of the protocol based on assessment of the
+            criteria above and communications with the team.
+          </li>
+          <li>
+            In the event of a security issue the methodologist will work with
+            the project team to understand the issue and any effects to $MVI
```

Comment:
I think it's possible the text content on this page will be updated once DATA methodologists provide that but just in case:

```suggestion
            the project team to understand the issue and any effects to $DATA
```